apps/server/lib/bootstrap.ts

```diff
@@ -156,28 +160,27 @@ export const bootstrap = async (context, options) => {
 	await webServer.start();
 
 	logger.info(context, 'Setting up cache');
-	const cache = new core.MemoryCache(environment.redis);
+	const cache = new Cache(environment.redis);
 	if (cache) {
 		await cache.connect();
 	}
 
 	// Instantiate a core instance that will handle DB operations
-	const backendOptions =
+	const backendOptions: PostgresBackendOptions =
 		options && options.database
 			? Object.assign({}, environment.database.options, options.database)
 			: environment.database.options;
-	const jellyfish = (await core.create(context, cache, {
-		backend: backendOptions,
-	})) as any as coreType.JellyfishKernel;
+
+	const jellyfish = await Kernel.withPostgres(context, cache, backendOptions);
```

Comment:
Here too

Reply:
> actually @edufschmidt this probably shouldn't be merged until we get the newest core into the worker and plugins.

Yeah, I was thinking the same thing. Didn't know you had a PR planned for the core, so let's wait until it's merged.